salt/cloud/deploy/bootstrap-salt.sh

```diff
@@ -2530,6 +2530,7 @@ install_debian_8_git_deps() {
         __REQUIRED_TORNADO="$(grep tornado "${__SALT_GIT_CHECKOUT_DIR}/requirements/base.txt")"
         if [ "${__REQUIRED_TORNADO}" != "" ]; then
             __apt_get_install_noinput python-tornado
+            __apt_get_install_noinput python-pycurl
```

Comment:
we should go through all of the requirements, as I came across some issues
with cloud requirements too
when doing a Solaris build

David Murphy
Unix Engineer
SaltStack

3400 North Ashton Blvd., Suite 110 | Lehi, UT 84043, USA

office 801-207-7440 | mobile 385-333-1060
dmurphy@saltstack.com |  www.saltstack.com
*SaltConf16 - Salt Lake City - April 19-21, 2016​ - www.saltconf.com
<http://www.saltconf.com/>​*



On Wed, Dec 2, 2015 at 9:07 AM, Justin Findlay <notifications@github.com>
wrote:

> In salt/cloud/deploy/bootstrap-salt.sh
> <https://github.com/saltstack/salt/pull/29322#discussion_r46433537>:
>
> > @@ -2530,6 +2530,7 @@ install_debian_8_git_deps() {
> >          __REQUIRED_TORNADO="$(grep tornado "${__SALT_GIT_CHECKOUT_DIR}/requirements/base.txt")"
> >          if [ "${__REQUIRED_TORNADO}" != "" ]; then
> >              __apt_get_install_noinput python-tornado
> > +            __apt_get_install_noinput python-pycurl
>
> @mrproper <https://github.com/mrproper>, bootstrap changes should be made
> at https://github.com/saltstack/salt-bootstrap. Salt bootstrap
> development happens somewhat independently from salt and bootstrap-salt.sh
> is regularly merged into the salt codebase.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/saltstack/salt/pull/29322/files#r46433537>.
>